include/openenclave/internal/syscall/declarations.h

```diff
@@ -256,6 +258,7 @@ OE_DECLARE_SYSCALL4_M(SYS_socketpair);
 #if __x86_64__ || _M_X64
 OE_DECLARE_SYSCALL2(SYS_stat);
 #endif
+OE_DECLARE_SYSCALL5(SYS_statx);
```

Comment:
Do we need statx? I don't see it being defined anywhere.
If statx is needed, add a comment here explaining why.